### ISSUE TITLE
Detect and autofix the usage of `id` property on the root level node in a XMLView

### ISSUE THREAD

**Issue by flovogt** [SAP/ui5-linter#1025]
I define a `sap/ui/core/mvc/XMLView` and specify the `id` property on the root level node. This property is not supported in the root level node and has no influence.

```xml
<mvc:View
  id="myFanyView"
  xmlns:mvc="sap.ui.core.mvc"
  xmlns="sap.m"
  controllerName="ui5app.controller.Home">
    <Text text="Hello"/>
<mvc:View>
```

# Detection Goal
This scenario is detected and a finding is reported to remove the useless `id` property.

# Autofix Goal
The `id` property is removed from the view definition.

```xml
<mvc:View
  xmlns:mvc="sap.ui.core.mvc"
  xmlns="sap.m"
  controllerName="ui5app.controller.Home">
    <Text text="Hello"/>
<mvc:View>
```

**Comment by flovogt** [comment]
- Detection will be implemented in CPOUI5FOUNDATION-1327
- Autofix will be implemented in CPOUI5FOUNDATION-1328
